src/hungergames/Loader.php

```diff
@@ -42,7 +42,6 @@ public function onEnable(){
                 $this->scriptManager = new HGAPIScriptManager($this);
                 $this->mapBackup = new MapBackup($this);
                 $this->signManager = new SignManager($this);
-                $this->getServer()->getCommandMap()->register("hg", new HGCommand($this));
```

Comment:
this is for the `/hg` command in game, without it, it won't work.

Reply:
my stupid bugfixes :D thx for correction :D